tropohelper/services.py

```diff
@@ -26,6 +36,51 @@ def create_firehose(stack, name, bucket_arn, buffering_seconds=300, buffering_si
             RoleARN="arn:aws:iam::347225174248:role/firehose_delivery_role")
```

Comment:
Just noticed this. We have to extract these from the code. We shouldn't have PatientPop specific stuff in here. Would involve a fix here to make it a parameter and a matching fix in the aws repo to where we've used this function.